src/main/java/pl/touk/sputnik/connector/gerrit/GerritFacadeBuilder.java

```diff
@@ -42,7 +42,7 @@ public HttpClientBuilder extend(HttpClientBuilder httpClientBuilder, GerritAuthD
             }
         });
 
-        return new GerritFacade(gerritApi, gerritPatchset);
+        return new GerritFacade(gerritApi, gerritPatchset, configuration);
```

Comment:
This is a good place to read configuration, build a CommentFilter and pass it to constructor.